src/sql/workbench/services/accountManagement/browser/accountManagementService.ts

```diff
@@ -254,6 +261,24 @@ export class AccountManagementService implements IAccountManagementService {
 		});
 	}
 
+	/**
+	 * Removes all registered accounts
+	 */
+	public removeAccounts(): Thenable<boolean> {
```

Comment:
Any reason to make this async?

and the self shouldn't be necessary here

Reply:
Yeah the self should go away. I just made it async since the other code in the file follows the same paradigm. Don't want to add too much variety that can throw people off.

Although I don't mind either way and can make it use async/promises if you think that's a better way of doing it.